frontend/src/Pages/StudentManagement.tsx

```diff
@@ -4,9 +4,7 @@ import useSWR from 'swr';
 import AuthenticatedLayout from '../Layouts/AuthenticatedLayout';
 import {
     ArrowPathRoundedSquareIcon,
-    PencilIcon,
-    TrashIcon,
-    UserPlusIcon
+    TrashIcon
 } from '@heroicons/react/20/solid';
```

Comment:
make sure to change this to `@heroicons/react/24/outline` (or really just move those icons into the import on line 30)